app/controllers/schedules_controller.rb

```diff
@@ -59,18 +59,26 @@ def edit
   end
 
   def create
-    @schedule = Schedule.new(schedule_params)
+    # Check to see if it has a root task before creating a schedule
+    msg = check_root_task(schedule_params[:tasks_attributes])
+    unless msg.nil?
+      flash[:notice] = msg
```

Comment:
Change this to `flash[:danger]`. The symbol used for flash messages corresponds to [CSS in Bootstrap](http://getbootstrap.com/css/#helper-classes) - see other instances of `flash` for more examples.